jsonpb/jsonpb_test.go

```diff
@@ -520,9 +522,11 @@ var marshalingTests = []struct {
 	{"BytesValue", marshaler, &pb.KnownTypes{Bytes: &types.BytesValue{Value: []byte("wow")}}, `{"bytes":"d293"}`},
 	{"required", marshaler, &pb.MsgWithRequired{Str: proto.String("hello")}, `{"str":"hello"}`},
 	{"required bytes", marshaler, &pb.MsgWithRequiredBytes{Byts: []byte{}}, `{"byts":""}`},
+	{"FieldMask", marshaler, &pb.FieldMaskWKT{FieldMask: &types.FieldMask{Paths: []string{"a", "b", "c_d"}}}, `{"fieldMask":"a,b,cD"}`},
```

Comment:
Curious, is the behaviour of `c_d` -> `cD` documented anywhere? I couldn't find anything.

Reply:
Check the display_name example here https://developers.google.com/protocol-buffers/docs/reference/csharp/class/google/protobuf/well-known-types/field-mask

Reply:
thanks!